.travis.yml

```diff
@@ -24,9 +24,11 @@ matrix:
     - php: nightly
       env:
         - DEPS=lowest
+        - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
```

Comment:
There is a new argument `--ignore-platform-req` which can be used to ignore specific requirements.

We probably should use `--ignore-platform-req=php` for nightly builds tho.

https://twitter.com/naderman/status/1277941647003848707

Reply:
Never mind. Thats a dedicated Composer 2 feature 👎